README.md

```diff
@@ -111,6 +112,57 @@ To execute the end-to-end tests affected by a change:
 yarn affected:e2e
 ```
 
+### Running System E2E tests
+
+To execute end-to-end tests via [Playwright](https://playwright.dev), run one or more files. The way to specify these, is similar as described in the docs [here](https://playwright.dev/docs/running-tests). 
```

Comment:
There's already docs for system e2e testing with Playwright here: https://docs.devland.is/repository/system-e2e

Let's merge this content into that and point to it from here.